### PR TITLE
MeteoSwiss extensions for OPS

### PR DESCRIPTION
**Pull request by dominichofer**
Integrates codes formally hosted in https://github.com/MeteoSwiss-APN/flexpart-opr/tree/balfrin-ifs

### CODE REVIEW

File: README.meteoswiss.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+Installation
+------------
+
+### Using the `spack` package
+
+Activate [spack-c2sm](https://github.com/C2SM/spack-c2sm/blob/main/README.md)
+and build flexpart with `spack`: 
+
+    spack dev-build -u build flexpart-ifs @develop %gcc
```
Comment (victoria-cherkas) [inline README.meteoswiss.md]: The flexpart-ifs spack package still points to the flexpart-ifs repo, not this one. 
Or does the dev-build instead use the curent repo as source? https://github.com/C2SM/spack-c2sm/blob/main/repos/c2sm/packages/flexpart-ifs/package.py

Comment (dominichofer) [reply README.meteoswiss.md]: Good point. We should mention that this requires at least spack-c2sm v0.20.1.2, which will be released next week and I'm currently working on.

File: README.meteoswiss.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+FLEXPART for IFS (and GFS) adapted source code
+for operational MeteoSwiss production.
+
+
+Installation
+------------
+
+### Using the `spack` package
+
```
Comment (dominichofer) [inline README.meteoswiss.md:12]:
```suggestion

This requires at lease spack-c2sm v0.20.1.2.
```

File: README.meteoswiss.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+This requires at least spack-c2sm v0.20.1.2.
+Activate [spack-c2sm](https://github.com/C2SM/spack-c2sm/blob/main/README.md)
+and build flexpart with `spack`: 
+
+    spack dev-build -u build flexpart-ifs @main
+
+### Using `make` (without `spack`)
+
+`makefile_meteoswiss` requires four environment variables:
+- ECCODES_DIR: The directory where eccodes is installe.
```
Comment (victoria-cherkas) [inline README.meteoswiss.md]:
```suggestion
- ECCODES_DIR: The directory where eccodes is installed.
```

**Pull request merged**